resources/drag_and_drop_helper.js

```diff
@@ -0,0 +1,53 @@
+(function( $ ) {
```

Comment:
Add comments on each function for better understanding in future

Reply:
already added